src/keyword.ml

```diff
@@ -0,0 +1,65 @@
+open! Import
+
+let is_keyword word =
+  let keywords =
+    [
+      "and";
+      "as";
+      "assert";
+      "begin";
+      "class";
+      "constraint";
+      "do";
+      "done";
+      "downto";
+      "else";
+      "end";
+      "exception";
+      "external";
+      "false";
+      "for";
+      "fun";
+      "function";
+      "functor";
+      "if";
+      "in";
+      "include";
+      "inherit";
+      "initializer";
+      "lazy";
+      "let";
+      "match";
+      "method";
+      "module";
+      "mutable";
+      "new";
+      "nonrec";
+      "object";
+      "of";
+      "open";
+      "or";
+      (*  "parser"; *)
+      "private";
+      "rec";
+      "sig";
+      "struct";
+      "then";
+      "to";
+      "true";
+      "try";
+      "type";
+      "val";
+      "virtual";
+      "when";
+      "while";
+      "with";
+      "lor";
+      "lxor";
+      "mod";
+      "land";
+      "lsl";
+      "lsr";
+      "asr";
+    ]
+  in
+  List.mem ~set:keywords word
```

Comment:
Given that we only use the list inside this function, we could write as follow:

```
let is_keyword = function
  | "and" | "as" | ... | "asr" -> true
  | _ -> false
```

which would make it much faster.

Reply:
Right, I didn't think about that. Thanks!